merlin/config/utils.py

```diff
@@ -8,17 +9,20 @@ class Priority(enum.Enum):
     mid = 2
     low = 3
 
-def is_rabbit_broker(broker):
+
+def is_rabbit_broker(broker: str) -> bool:
     return broker in ["rabbitmq", "amqps", "amqp"]
 
-def is_redis_broker(broker):
+
+def is_redis_broker(broker: str) -> bool:
     return broker in ["redis", "rediss", "redis+socket"]
 
-def get_priority(priority):
-    broker = CONFIG.broker.name.lower()
-    priorities = [Priority.high, Priority.mid, Priority.low]
-    if priority not in priorities:
-        raise ValueError(
+
+def get_priority(priority: Priority) -> int:
```

Comment:
this is called eg via get_priority(Priority.high) or get_priority(Priority.low). Are these Priority objects? Enums are weird sometimes.

Reply:
Yeah, they can be funky. If passed that way it'll be a Priority, although if it were Priority.high.name or Priority.high.val it'd be a str or int. Could more broadly type it as an Enum, but that would actually fail on something like `assert type(priority_obj) == Enum)` (you'd have to use isinstance() to get the parent Enum for Priority). Generally safer to be as specific as possible unless you have a specific reason to be generic (like a factory method needing to handle different classes).